packages/pipeline-editor/src/properties.json

```diff
@@ -298,7 +298,7 @@
               "condition": {
                 "parameter_ref": "gpu",
                 "op": "greaterThan",
-                "value": 0
+                "value": -1
```

Comment:
The `validation.fail_message.message.default` field above should read to the effect of "GPU must be greater than or equal to 0."

Reply:
oh yep, good catch